eventhubs/client/azure-eventhubs/src/main/java/com/azure/core/amqp/AmqpLink.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import java.io.Closeable;
+
+/**
+ * Represents a unidirectional AMQP link.
+ */
+public interface AmqpLink extends Closeable {
+    /**
+     * Gets the name of the link.
+     *
+     * @return The name of the link.
+     */
+    String linkName();
```

Comment:
There is a naming inconsistency in some of your interfaces between the fluent style `linkName()` and, e.g. `getHost()`. I'm leaning towards the spec stating more clearly that the fluent style naming pattern should only be used in models and where `this` is returned. In all other cases, we should use the standard `get`, `set`, and `is` JavaBean naming pattern.